build.sh

```diff
@@ -42,15 +52,15 @@ function test_docker_container() {
     grep -E "BROKEN|Getting links from" | \
     grep -B 1 "BROKEN";
 
-  echo "Done checking links"
+  builder_echo blue "Done checking links"
   rm tier.txt
   return "${BLC_RESULT}"
 }
 
 builder_run_action configure  bootstrap_configure
 builder_run_action clean      clean_docker_container $KEYMAN_IMAGE_NAME $KEYMAN_CONTAINER_NAME
 builder_run_action stop       stop_docker_container  $KEYMAN_IMAGE_NAME $KEYMAN_CONTAINER_NAME
-builder_run_action build      build_docker_container $KEYMAN_IMAGE_NAME $KEYMAN_CONTAINER_NAME
+builder_run_action build      build_docker_container $KEYMAN_IMAGE_NAME $KEYMAN_CONTAINER_NAME $BUILDER_CONFIGURATION
```

Comment:
This passes in the BUILDER_CONFIGURATION parameter for keymanapp/shared-sites#84